pkg/apiserver/api_installer.go

```diff
@@ -139,7 +139,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	// we're trying to register here
 	fqKindToRegister := unversioned.GroupVersionKind{}
 	for _, fqKind := range fqKinds {
-		if fqKind.Group == a.group.GroupVersion.Group {
+		if fqKind.Group == a.group.GroupVersion.Group || (fqKind.Group == extensions.GroupName && fqKind.Kind == "ScaleTwo") {
```

Comment:
we don't want to special case exceptions like this. we either want to allow configured overrides for specific subresources, or remove the restriction of the submitted group/version matching the resthandler's group version as long as it could be converted to it